### ISSUE TITLE
A plugin to let you control mount music per mount

### ISSUE THREAD

**Issue by EpicNinjaGuy123** [goatcorp/suggestions#1047]
### Plugin Description

Many mounts share the same default music, which could be replaced for a more appropriate track. This plugin would allow you to set specific music for every mount in the game, so you dont get the same music 50% of the time you click mount roulette.

### Request Terms

- [X] I checked if this request wasn't already proposed
- [X] I made sure this request follows Dalamud's [plugin guidelines](https://github.com/goatcorp/FFXIVQuickLauncher)

**Comment by Sohtoren** [comment]
AutoMountBGM does that

**Comment by roflqu** [comment]
You're mistaken. AutoMountBGM determines whether you're going to have the default Mount music or the music for the area you're in playing. It doesn't let you choose songs to play. It'd be very nice if we could have this particular plugin.
